validator/tests/test_scheduler/tests.py

```diff
@@ -43,7 +43,55 @@
 
 class TestSchedulersWithYaml(unittest.TestCase):
 
-    def test_single_block_files_individually(self):
+    def _setup_serial_scheduler(self):
+        context_manager = self._context_manager
+        squash_handler = context_manager.get_squash_handler()
+        first_state_root = context_manager.get_first_root()
+        scheduler = SerialScheduler(squash_handler,
+                                    first_state_root,
+                                    always_persist=False)
+        return context_manager, scheduler
+
+    def _setup_parallel_scheduler(self):
+        context_manager = self._context_manager
+        squash_handler = context_manager.get_squash_handler()
+        first_state_root = context_manager.get_first_root()
+        scheduler = ParallelScheduler(squash_handler,
+                                      first_state_root)
+        return context_manager, scheduler
+
+    def test_parallel_simple_scheduler_test(self):
```

Comment:
Yeah, and what makes it more complicated is there is a yaml file in test_scheduler/data that goes along with each test. I made the yaml filename part of the test method name, so 'test_parallel_simple_scheduler_test' is the parallel scheduler being run against the 'simple_scheduler_test.yaml' file. Do you have any thoughts on how to make it clearer?